rllab/envs/gym_env.py

```diff
@@ -86,7 +90,10 @@ def __init__(self, env_name, record_video=True, video_schedule=None, log_dir=Non
         logger.log("observation space: {}".format(self._observation_space))
         self._action_space = convert_gym_space(env.action_space)
         logger.log("action space: {}".format(self._action_space))
-        self._horizon = env.spec.tags['wrapper_config.TimeLimit.max_episode_steps']
```

Comment:
Found the root cause of this issue here:
https://github.com/openai/gym/blob/master/gym/envs/registration.py#L48

So it seems like the real fix here is to bump the OpenAI gym version and use env.max_episode_steps. My local repo uses the latest version of gym, so I don't think bumping it (in environment.yml) should cause problems.